subsys/bluetooth/mesh/scene_srv.c

```diff
@@ -155,7 +177,7 @@ static int scene_register_status_send(struct bt_mesh_scene_srv *srv,
 					 2 * CONFIG_BT_MESH_SCENES_MAX);
 	bt_mesh_model_msg_init(&buf, BT_MESH_SCENE_OP_REGISTER_STATUS);
 	net_buf_simple_add_u8(&buf, status);
-	net_buf_simple_add_le16(&buf, srv->curr);
+	net_buf_simple_add_le16(&buf, current_scene(srv, k_uptime_get()));
```

Comment:
Looks like this is an existing bug, but 5.1.3.2.1 has this sentence:

> When a scene transition is in progress, the value of the Current Scene state shall be set to 0x0000.

This makes no sense for the scene_status message, but it might make sense here?

Reply:
Why do it not make sense for the scene_status message? I would say it makes more sense for the status message. But does this apply for both in within delay and transition?